bin/internal/dart-sdk.version

```diff
@@ -1 +1 @@
-1.25.0-dev.7.0
+hash/a83a69ff3eff845980308779cadbdb80cc03a603
```

Comment:
As discussed, this issue should be resolved by the kernel/front-end providing a stable interface.